start-mailserver.sh

```diff
@@ -173,6 +173,26 @@ case $DMS_SSL in
 
 esac
 
+if [ -f /tmp/postfix/main.cf ]; then
+  while read line; do
+    postconf -e "$line"
```

Comment:
Could you add an integration test on this, like overriding a default value.
Add a `test/main.cf` with one override. Do not forget to update `make clean` task in the `Makefile`.